packages/ui/components/CssBaseLine/CssBaseLine.tsx

```diff
@@ -1,3 +1,25 @@
 import { CssBaseline as MuiCssBaseline } from "@mui/material";
+import { makeStyles, createStyles } from "@mui/styles";
 
-export const CssBaseLine = MuiCssBaseline;
+const useGlobalStyles = makeStyles(() =>
+  createStyles({
+    "@global": {
+      a: {
+        color: "inherit",
+        textDecoration: "none",
+      },
+      "*": {
+        boxSizing: "border-box",
+      },
+      "html, body, #root": {
+        padding: 0,
+        margin: 0,
+      },
+    },
+  })
+);
+
+export const CssBaseLine = () => {
+  useGlobalStyles();
+  return <MuiCssBaseline />;
+};
```

Comment:
Adding in some more global styles, just stuff that was in individual `global.css` files – which I removed here.